clang/include/clang/Basic/Attr.td

```diff
@@ -1263,15 +1263,17 @@ def SYCLRegisterNum : InheritableAttr {
   let Documentation = [SYCLRegisterNumDocs];
 }
 
-// Used by FE to mark ESIMD kernel pointer parameters which correspond to the
+// Used by FE to mark SYCL kernel pointer parameters which correspond to the
 // original lambda's captured accessors. FE turns the attribute to some metadata
-// required by the ESIMD Back-End.
-// Not supposed to be used directly in the source - SYCL device compiler FE
-// automatically adds it for ESIMD kernels, hence undocumented.
-def SYCLSimdAccessorPtr : InheritableAttr {
-  // No spelling, as this attribute can't be created in the source code.
+// required by the FPGA back-end.
+// This attribute does not require custom semantic handling 
+// hence we set the SemaHandler field to 0.
```

Comment:
I do not think you need to mention this here as we already have inside Attr.td file
https://github.com/intel/llvm/blob/c31d93ec2379f6d9b15309255ec51e4061c1b979/clang/include/clang/Basic/Attr.td#L534-#L535